docs/libraries/mst-form-type.md

```diff
@@ -0,0 +1,95 @@
+# MST Form Type
+
+npm: https://www.npmjs.com/package/mst-form-type
+
+## Introduction
+
+Librarys like Ant Design has a build-in Form component that can hold fields status and validation rules. This component houses its own store to sustain internally related field statuses and can be conveniently used independently. integrating Form component with MobX State Tree models can pose significant challenges as business logic grows intricately complex.
```

Comment:
**question:** can you please list a few examples of those challenges? It would be great to articulate more about why the `Form` component is hard to use with MST models.